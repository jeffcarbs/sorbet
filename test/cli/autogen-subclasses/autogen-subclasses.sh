#!/bin/bash
set -eu

echo "--- autogen-subclasses ---"
main/sorbet --silence-dev-message --stop-after=resolver -p autogen-subclasses --autogen-subclasses-parent=Opus::Mixin --autogen-subclasses-parent=Opus::Parent test/cli/autogen-subclasses/a.rb
