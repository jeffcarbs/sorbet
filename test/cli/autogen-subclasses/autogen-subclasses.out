--- autogen-subclasses ---
Opus::Mixin
 Opus::Mixed
 Opus::MixedDescendant
Opus::Parent
 Opus::Child
 Opus::DupChild
